runconfig/opts/parse.go

```diff
@@ -22,88 +22,87 @@ import (
-	flNetMode           string
-	flMacAddress        string
-	flIPv4Address       string
-	flIPv6Address       string
-	flIpcMode           string
-	flPidsLimit         int64
-	flRestartPolicy     string
-	flReadonlyRootfs    bool
-	flLoggingDriver     string
-	flCgroupParent      string
-	flVolumeDriver      string
-	flStopSignal        string
-	flIsolation         string
-	flShmSize           string
-	flNoHealthcheck     bool
-	flHealthCmd         string
-	flHealthInterval    time.Duration
-	flHealthTimeout     time.Duration
-	flHealthRetries     int
-	flRuntime           string
-	flAutoRemove        bool
+	Attach            opts.ListOpts
+	Volumes           opts.ListOpts
+	Tmpfs             opts.ListOpts
+	BlkioWeightDevice WeightdeviceOpt
+	DeviceReadBps     ThrottledeviceOpt
+	DeviceWriteBps    ThrottledeviceOpt
+	Links             opts.ListOpts
+	Aliases           opts.ListOpts
+	LinkLocalIPs      opts.ListOpts
+	DeviceReadIOps    ThrottledeviceOpt
+	DeviceWriteIOps   ThrottledeviceOpt
+	Env               opts.ListOpts
+	Labels            opts.ListOpts
+	Devices           opts.ListOpts
+	Ulimits           *UlimitOpt
+	Sysctls           *opts.MapOpts
+	Publish           opts.ListOpts
+	Expose            opts.ListOpts
+	DNS               opts.ListOpts
+	DNSSearch         opts.ListOpts
+	DNSOptions        opts.ListOpts
+	ExtraHosts        opts.ListOpts
+	VolumesFrom       opts.ListOpts
+	EnvFile           opts.ListOpts
+	CapAdd            opts.ListOpts
+	CapDrop           opts.ListOpts
+	GroupAdd          opts.ListOpts
+	SecurityOpt       opts.ListOpts
+	StorageOpt        opts.ListOpts
+	LabelsFile        opts.ListOpts
+	LoggingOpts       opts.ListOpts
+	Privileged        bool
+	PidMode           string
+	UTSMode           string
+	UsernsMode        string
+	PublishAll        bool
+	Stdin             bool
+	Tty               bool
+	OomKillDisable    bool
+	OomScoreAdj       int
+	ContainerIDFile   string
+	Entrypoint        string
+	Hostname          string
+	MemoryString      string
+	MemoryReservation string
+	MemorySwap        string
+	KernelMemory      string
+	User              string
+	WorkingDir        string
+	CPUShares         int64
+	CPUPercent        int64
+	CPUPeriod         int64
+	CPUQuota          int64
+	CpusetCpus        string
+	CpusetMems        string
+	BlkioWeight       uint16
+	IOMaxBandwidth    string
+	IOMaxIOps         uint64
+	Swappiness        int64
+	NetMode           string
+	MacAddress        string
+	IPv4Address       string
+	IPv6Address       string
+	IpcMode           string
+	PidsLimit         int64
+	RestartPolicy     string
+	ReadonlyRootfs    bool
+	LoggingDriver     string
+	CgroupParent      string
+	VolumeDriver      string
+	StopSignal        string
+	Isolation         string
+	ShmSize           string
+	NoHealthcheck     bool
+	HealthCmd         string
+	HealthInterval    time.Duration
+	HealthTimeout     time.Duration
+	HealthRetries     int
+	Runtime           string
+	AutoRemove        bool
```

Comment:
These have changed from unexported to exported. I think they should be lowercase to keep them unexported.